src/app/client/src/app/modules/observation/guards/ml/ml.guard.ts

```diff
@@ -24,6 +24,10 @@ export class MlGuard implements CanActivate {
         return true
       } else {
         this.toasterService.error(_.get(this.resourceService, 'messages.stmsg.m0145'));
+        let queryParam = {
```

Comment:
if it is not changing please make it a const and as it is object it can be const
and you can change its attributes